contracts/pricecheckers/SingleSidedBPTExpectedOutCalculator.sol

```diff
@@ -0,0 +1,1243 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+// File: BalancerErrors.sol
+
+// This program is free software: you can redistribute it and/or modify
```

Comment:
You can import the fixedpoint lib like so: `import "@balancer/lib/math/FixedPoint.sol";`